pkg/cmd/build/build.go

```diff
@@ -71,5 +73,17 @@ func Run(ctx context.Context, namespace, buildKitHost string, isOktetoCluster bo
 		analytics.TrackBuildTransientError(buildKitHost, success)
 		return err
 	}
+	if err != nil {
+		imageRegistry, imageTag := registry.GetRegistryAndRepo(tag)
+		if registry.IsLoggedIntoRegistryButDontHavePermissions(err) {
+			err = okErrors.UserError{E: fmt.Errorf("You are not authorized to push image '%s'.", imageTag),
+				Hint: fmt.Sprintf("Please login into the registry '%s' with a user with write permissions to '%s' or use another image.", imageRegistry, imageTag)}
```

Comment:
@jLopezbarb @pchico83  in a docker registry, the permissions are called pull/push, not read/write. Can we update the message to use that terminology?